doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -48,6 +48,7 @@ features:
   wifi:
     STA Mode: WPA_SUPP && WIFI_NRF700X
     Scan only (for location accuracy): WIFI_NRF700X && !WPA_SUPP
+    SR Co-existence: WIFI_NRF700X && NRF700X_BT_COEX
```

Comment:
Should the 802.15.4 COEX also be added here?

Reply:
@jciupis

Reply:
It's a difficult question to answer. First of all, the change introduced in this PR is misleading because it implies that Short Range Coexistence is only about BLE, which is incorrect. Since to my best knowledge coexistence with 802.15.4 on PHY level can be considered production quality, we could include it in the list here.

However, we know about issues that prevent us from claiming production quality on Thread + WiFi coexistence. Claiming support for WiFi coexistence with 802.15.4 might be misleading if customers assume that it implies also WiFi coexistence with Thread. That could be a reason to _not_ include 802.15.4 in the list here.

Finally, if we decide to mention 802.15.4 after all we could either add it and BLE separately or treat it as Short Range and share the documentation. Coexistence is handled by MPSL both for BLE and for 802.15.4, so it could be misleading for customers to treat them separately as if they were different solutions. At the same time, there might be differences in support on higher layers between BLE and 802.15.4, so it could also make sense to treat them separately.

I don't fully understand what the change in this PR is supposed to document, so I'm struggling to recommend any solution. @krish2718 could you please explain what you're documenting with this change?

Reply:
The idea was to add Wi-Fi - BLE co-existence but as we are calling it by the generic name SR coex (even the sample is named that way `samples/wifi/sr_coex`) I have used that. IMHO, we can use BLE coex for now, and add separate entries for 802.15.4 + Wi-Fi and Thread + Wi-Fi.